src/DDoSX/DomainClient.php

```diff
@@ -99,7 +102,42 @@ public function getIpByName($domainName)
         $response = $this->request("GET", 'v1/domains/' . $domainName . '/ips');
         $body = $this->decodeJson($response->getBody()->getContents());
 
-        return new Ip($this->apiToFriendly($body->data, $this->ipMap));
+        return new Ip($this->apiToFriendly($body->data, self::IP_MAP));
+    }
+
+    /**
+     * Get a paginated list of domain properties
+     * @param $domainName
+     * @param int $page
+     * @param int $perPage
+     * @param array $filter
+     * @return int|\UKFast\SDK\Page
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function getProperties($domainName, $page = 1, $perPage = 15, $filter = [])
```

Comment:
DDoSX uses 20 as a default:

```suggestion
    public function getProperties($domainName, $page = 1, $perPage = 20, $filter = [])
```

Reply:
Actually, in one of your other MRs, this is a different client, should we move this to that client?